### ISSUE TITLE
bash history

### ISSUE THREAD

**Issue by atselousov** [neuromation/cookiecutter-neuro-project#332]
Guys, is it possible to add saving `bash` history between runs of `develop` jobs from the box? For example, create a file with history on storage in a project folder during `setup` and mount this file to file system in develop jobs.

**Comment by ayushkovskiy** [comment]
Good idea! Though, we need to be very careful there. From security perspective, it might be a problem  if the user's bash history file is sent to the Platform without asking for permission as it might contain sensitive information. I would think of it more carefully and if needed, implement this feature with a trigger-flag disabled by default.

**Comment by YevheniiSemendiak** [comment]
I see one possible implementation, yet its super-ugly:
1. after starting the job, copy bash history file from the storage
2. update bash history file on storage:
3. during the job run - sync the file with the storage. 

This leads to:
1. `develop` job singleton. we cannot dump the updates from several jobs.
2. obligatory mount of project folder into the job with R/W mode
Not everybody will be sattisfied with that.

Do you need that @atselousov?
If yes, maybe it's a matter of docs. We might construct a config file for you, with help of which you will be able to sync bash history between job runs.

**Comment by YevheniiSemendiak** [comment]
@atselousov, @mariyadavydova, another idea came into my mind.

What if we add a [mixin](https://neu-ro.gitbook.io/neuro-flow/reference/mixins), which will:
- use a volume (or even define, in future) for storing bash/zsh history + *.rc files
- point to the local bash/zsh files 
- `upload(volume)` to the storage on job startup and mount it to the job's `/root` or `/home/user` folder
